db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The 'bonus' field in the Skill model is defined as a CharField. If this field is intended to store numerical values, consider using an IntegerField or a FloatField instead. This will ensure data integrity and allow for numerical operations.